change/office-ui-fabric-react-2019-12-03-20-40-54-grouped-list-recursive-height.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "major",
```

Comment:
This change type should be `patch` since no API signature was changed.

Reply:
Got it, I'm changing it to `patch` .

I was not sure whether to classify this as a breaking change, in theory there could be applications that are dependent on the flawed behavior of `_getGroupHeight`. It seems highly unlikely though.